streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorNodeTest.java

```diff
@@ -156,8 +156,7 @@ public void testTopologyLevelConfigException() {
 
         final ConfigException se = assertThrows(ConfigException.class, () -> new TopologyTestDriver(topology));
         final String msg = se.getMessage();
-        assertTrue("Error about class cast with serdes", msg.contains("StreamsConfig#DEFAULT_KEY_SERDE_CLASS_CONFIG"));
```

Comment:
Relates to my other commend -- preserving the original message, which tells the user a relevant config seems useful, and we should keep it?

Reply:
recovered test